### ISSUE TITLE
Remove morty and filtron leftovers

### ISSUE THREAD

**Issue by return42** [searxng/searxng#3888]
Morty and filtron are unmaintained components and had been replaced by an image-proxy and the bot-detection in SearXNG since years .. we should remove any leftovers:

Related:

- [The infrastructure of SearXNG has changed dramatically over the past year. The installation scripts and documentation have not been adapted so far.](https://github.com/searxng/searxng/pull/1332)
- filtron https://github.com/search?q=repo%3Asearxng%2Fsearxng+filtron+&type=code
- morty https://github.com/search?q=repo%3Asearxng%2Fsearxng+morty+&type=code

**Comment by allixx** [comment]
what is the alternative for morty as result_proxy service? occasionally I like to open search results via morty. image_proxy does not provide this functionality.

for what is worth, morty works just fine on private instances.

**Comment by Bakr-Ali** [comment]
> what is the alternative for morty as result_proxy service? occasionally I like to open search results via morty. image_proxy does not provide this functionality.
> 
> for what is worth, morty works just fine on private instances.

Did you find any alternative? Do you know where to find a list of searxng instances that still support morty proxy?

I don't understand why anyone would remove such a useful feature

**Comment by return42** [comment]
> I don't understand why anyone would remove such a useful feature

Did you ever tested .. it does not work with most modern WEB sites and morty is not maintained .. last release was [in 2018](https://github.com/asciimoo/morty/tags) .. users gave up in 2022 to report issues .. --> nobody really uses it anymore.

**Comment by Bakr-Ali** [comment]
> Did you ever tested .. it does not work with most modern WEB sites 

Yes, I did (on the only working instance i know of), and it works perfectly as far as I have tested on multiple websites and without any errors.
Example: https://morty.ononoki.org/?mortyurl=https://www.euronews.com/next/2025/02/26/asteroid-2024-yr4-is-no-longer-a-threat-to-earth-scientists-say

> and morty is not maintained .. last release was [in 2018](https://github.com/asciimoo/morty/tags) .. users gave up in 2022 to report issues .. --> nobody really uses it anymore.

I didn't notice that until after I wrote my comment here.

Do you know of any alternatives / similar services?

**Comment by loorisr** [comment]
maybe https://www.privoxy.org/ or https://mitmproxy.org can be a replacement?
